packages/firestore/README.md

```diff
@@ -42,3 +42,7 @@ For comprehensive documentation please see the [Firebase Reference
 Docs][reference-docs].
 
 [reference-docs]: https://firebase.google.com/docs/reference/js/
+
+## Contributing
+See [Contributing](./CONTRIBUTING.md) for more information on contributing to
```

Comment:
It may be worth linking to the top-level `CONTRIBUTING.md` here, indicating that it's contains how to set up the environment, etc while this file is about how to run tests and stuff like that.